tests/integration_tests.rs

```diff
@@ -1,14 +1,21 @@
 extern crate assert_cmd;
 
+mod tester;
+
 use assert_cmd::prelude::*;
 use std::process::Command;
+use tester::BatTester;
 
 fn bat() -> Command {
     let mut cmd = Command::main_binary().unwrap();
     cmd.current_dir("tests/examples");
     cmd
 }
 
+fn tester() -> BatTester {
```

Comment:
This seems a little confusing besides `fn bat()`. Perhaps should I rename it to `fn snapshot_tester()`?